### PR TITLE
MES-915 - Add build script to deploy onto device

### PR DESCRIPTION
**Pull request by johnmorleybjss**
* End goal is to produce a `.ipa` file that can be shipped to Capita to be deployed to Airwatch
* Fixes issue with Cordova build by bumping version of `cordova-plugin-ionic-webview`
* Provides npm script `device` which deploys to USB connected device).

### CODE REVIEW

**Comment by johnmorleybjss** [comment]
After feedback from reviewers (not in this thread), the decision has been made to not have any artifact build scripts. The `package` script has therefore been removed.

File: buildScripts/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Provisioning Profile
+
+In order to create a provisioning profile for development, follow these steps:
+
+1.  Run `ionic cordova build ios` to generate an XCode project.
+2.  Open the workspace at `platforms/ios/Mobile Examiner.xcworkspace` with XCode.
+3.  In XCode, goto Preferences (`Cmd+comma`) -> Accounts => Add (plus symbol, dialog lower left) and sign in with your Apple ID.
+4.  Close the Preferences dialog. In the top left of the main XCode window select the icon that looks like a folder, then the project name.
+5.  Ensure your "Bundle Identifier" is unique (you can probably just suffix your initials).
+6.  Ensure "Automatically manage signing" is selected. In the "Team" dropdown, select your personal team. A Provisioning Profile should be populated.
```
Comment (JamieKellyBJSS) [inline buildScripts/README.md]:
```suggestion
6.  Select a connected device it the top menu bar.
7.  Ensure "Automatically manage signing" is selected. In the "Team" dropdown, select your personal team. A Provisioning Profile should be populated.
8.  Determine your `TeamIdentifier` in your provisioning profile (found at `~/Library/MobileDevice/Provisioning Profiles`).
```

File: buildScripts/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+7.  Determine your `TeamIdentifier` in your provisioning profile (found at `~/Library/MobileDevice/Provisioning Profiles`).
```
Comment (JamieKellyBJSS) [inline buildScripts/README.md]:
```suggestion
8.  Determine your `TeamIdentifier` in your provisioning profile (found at `~/Library/MobileDevice/Provisioning Profiles`).
```

**Comment by JamieKellyBJSS** [comment]
I'm happy with the instructions, but I can't build for some reason on my machine. All the instructions ran fine for me though, so I don't think the issue is in this MR.

**Comment by JamieKellyBJSS** [comment]
error: archive not found at path '/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/Mobile Examiner.xcarchive'
** EXPORT FAILED **

CordovaError: Promise rejected with non-error: 'Error code 65 for command: xcodebuild with args: -exportArchive,-archivePath,Mobile Examiner.xcarchive,-exportOptionsPlist,/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/exportOptions.plist,-exportPath,/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/build/device'

**Comment by ammarhaiderbjss** [comment]
> error: archive not found at path '/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/Mobile Examiner.xcarchive'
> ** EXPORT FAILED **
> 
> CordovaError: Promise rejected with non-error: 'Error code 65 for command: xcodebuild with args: -exportArchive,-archivePath,Mobile Examiner.xcarchive,-exportOptionsPlist,/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/exportOptions.plist,-exportPath,/Users/jamie.kelly/mes/mes-mobile-app/platforms/ios/build/device'

I've been getting this error too but the app was still building successfully, although I'm not getting it anymore on develop.

File: buildScripts/README.md (hunk 1 above)
Comment (johnmorleybjss) [reply buildScripts/README.md]: Made this changed.

File: buildScripts/README.md (hunk 2 above)
Comment (johnmorleybjss) [reply buildScripts/README.md]: Made this change.

**Comment by johnmorleybjss** [comment]
I've just picked this up again and also experienced the error above. I was able to alleviate it by going into XCode and configuring the usage of the legacy build system (see 83211b5533f2858febbbd7aef9db61a56e315680). I'm not sure why this manual step is required, I thought it would have been covered by `-UseModernBuildSystem=0` in the `build.json`. I have put an extra step in the README.

**Comment by johnmorleybjss** [comment]
Reverting the name after feedback.

**Pull request merged**